docs/basic-config.asciidoc

```diff
@@ -255,6 +255,10 @@ const client = new Client({
 |`boolean`, `'proto'`, `'constructor'` - By the default the client will protect you against prototype poisoning attacks. Read https://web.archive.org/web/20200319091159/https://hueniverse.com/square-brackets-are-the-enemy-ff5b9fd8a3e8?gi=184a27ee2a08[this article] to learn more. If needed you can disable prototype poisoning protection entirely or one of the two checks. Read the `secure-json-parse` https://github.com/fastify/secure-json-parse[documentation] to learn more. +
 _Default:_ `false`
 
+|`caFingerprint`
```

Comment:
__nit:__ I agree with @thomheymann that it'd make sense to mention that we're expecting SHA-256 digest - otherwise consumer may not realize why their SHA-1 fingerprint doesn't work.

We could also easily check if provided SHA-1 or SHA-256 is valid (based on the predetermined string length and knowing that it's a HEX string) and throw if it doesn't pass validation, but I'll leave to you :slightly_smiling_face: